project.js

```diff
@@ -52,69 +49,121 @@ const projects = [
+        <li>Back End Dev</li>
+        <li>2015</li>
       </ul>
+    </div>
+        <p class="card1-description-p">
+        ${selectedProject.description}
+        </p>
+    
+    <ul class="language">
+      <li>${selectedProject.technologies[0]}</li>
+      <li>${selectedProject.technologies[1]}</li>
+      <li>${selectedProject.technologies[2]}</li>
+    </ul>
+    <button type="button" class="pro-button" id="1">${selectedProject.link_to_live[0]}</button>
 
-      <div class="project-detail-buttons">
-      <a  href="${selectedProject.link_to_live[2]}"><button class="work-button_1 see-source">${selectedProject.link_to_live[0]}
-<img class="see-source-img" src=${selectedProject.link_to_live[1]} alt="" /></button>
-</a>
-      <a href="${selectedProject.link_to_source[2]}"><button class="work-button_1 see-source">${selectedProject.link_to_source[0]}
-<img class="see-source-img"  src=${selectedProject.link_to_source[1]}  padding-top=30px ; alt="" /></button>
-</a>
-  </div>
-  </div>
-  </div>
+</div>
+</div>
+</div>
 
+<div class="card-container">
+  <div class="card">
+    <img src="${selectedProject1.featured_image}" class="cardpic" alt="" />
+   <div class="desktop">
+    <h2 class="card-title">${selectedProject1.name}</h2>
+    <h2 class="card-title-two">${selectedProject1.name}</h2>
+    <div class="card-about">
+      <p>CANOPY</p>
+      <ul>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+    </div>
+        <p class="card1-description-p">
+        ${selectedProject1.description}
+        </p>
     
-  </section>
+    <ul class="language">
+      <li>${selectedProject1.technologies[0]}</li>
+      <li>${selectedProject1.technologies[1]}</li>
+      <li>${selectedProject1.technologies[2]}</li>
+    </ul>
+    <button type="button" class="pro-button" id="2">${selectedProject.link_to_live[0]}</button>
+
 </div>
 </div>
 </div>
-  `;
 
-    const projectDetailClose = document.querySelector('.close-btn');
-    projectDetailClose.addEventListener('click', () => {
-      mainTag.classList.remove('fixed_postion');
-      header.classList.remove('fixed_postion');
-      projectDetail.classList.add('hide');
-    });
-  });
-});
+<div class="card-container">
+  <div class="card">
+    <img src="${selectedProject2.featured_image}" class="cardpic" alt="" />
+   <div class="desktop">
+    <h2 class="card-title">${selectedProject2.name}</h2>
+    <h2 class="card-title-two">${selectedProject2.name}</h2>
+    <div class="card-about">
+      <p>CANOPY</p>
+      <ul>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+    </div>
+        <p class="card1-description-p">
+        ${selectedProject2.description}
+        </p>
+    
+    <ul class="language">
+      <li>${selectedProject2.technologies[0]}</li>
+      <li>${selectedProject2.technologies[1]}</li>
+      <li>${selectedProject2.technologies[2]}</li>
+    </ul>
+    <button type="button" class="pro-button" id="3">${selectedProject2.link_to_live[0]}</button>
```

Comment:
- [ ] You've done a great job creating this section dynamically 👏 But we need to follow the Figma design. So please make sure this button has a `See project` like the Figma design 👇 
![Screenshot from 2022-06-25 17-12-44](https://user-images.githubusercontent.com/87965534/175777726-0039c4c9-b124-45c8-beab-ab6c1cb1231a.png)
Your design 👇 
![Screenshot from 2022-06-25 17-12-54](https://user-images.githubusercontent.com/87965534/175777742-f3e25c3d-f315-4bbf-b7af-0bfb5307af57.png)
- [ ] [OPTIONAL] Same here the buttons on the Figma design are next to each others👇 
Please fix yours to match it.
![Screenshot from 2022-06-25 17-13-37](https://user-images.githubusercontent.com/87965534/175778063-b29d7645-d207-4a49-b27f-995b463cd4dd.png)
Your design 👇 
![Screenshot from 2022-06-25 17-13-05](https://user-images.githubusercontent.com/87965534/175778129-b9e3d9ad-48e0-46cf-8d65-a6648511aee3.png)

Reply:
thank you for the review, I fixed it can you please review it again,

Reply:
@Sahar-AbdelSamad please i have pushed the changes can you please check for me I have only a few hours before the deadline

Reply:
@milen-ale It looks good 👍 good job.
KIndly ask for a new review from your dashboard. I am not authorized to approve it from here.